count.d.ts

```diff
@@ -0,0 +1,146 @@
+interface Window {
+	/**
+	 * GoatCounter JavaScript API
+	 *
+	 * @see https://www.goatcounter.com/help/js
+	 */
+	goatcounter:
+		| (GoatCounter.Settings &
+				GoatCounter.DataParameters &
+				GoatCounter.Methods)
+		| undefined;
```

Comment:
`undefined` accounts for scenarios in which the async script might still be loading.